src/components/button/Button.js

```diff
@@ -89,8 +89,6 @@ export class ButtonComponent extends BaseComponent {
           });
           break;
         case 'custom':
-          // Get the FormioForm at the root of this component's tree
-          var form = this.getRoot();
           // Get the form's flattened schema components
           var flattened = FormioUtils.flattenComponents(form.component.components, true);
```

Comment:
Won't ```form``` be undefined now?

Reply:
No. It is passed in to new Function below.

Reply:
Right, but in this line it calls.

```
var flattened = FormioUtils.flattenComponents(form.component.components, true);
```

which refers to a variable called ```form```. Won't that now be undefined since you removed the variable?

Reply:
Oh, yup. That's a problem